.github/workflows/prod-build.yml

```diff
@@ -222,6 +222,7 @@ jobs:
           CONTENT_TRANSLATED_ROOT: ${{ github.workspace }}/mdn/translated-content/files
 
           DEPLOYER_BUCKET_NAME: mdn-content-prod
+          DEPLOYER_DISTRIBUTION_ID: E2ZY2DGUN70EMI
```

Comment:
Let's comment this out for now. And only get this to stage.

Reply:
✅  in d78c7483e.